main.py

```diff
@@ -539,41 +553,48 @@ async def list_update():
     
     okib_list_string = ", ".join([member.display_name for member in _okib_members])
     noib_list_string = ", ".join([member.display_name for member in _noib_members])
-    _list_content = "{} asks:\n{} {}/{} : {}\n{} : {}".format(
-        _gatherer.display_name,
+    _list_content = "{} asks : {}\n{} {}/{} : {}\n{} : {}".format(
+        _gatherer.display_name,OKIB_GATHER_EMOJI_STRING,
         OKIB_EMOJI_STRING, len(_okib_members), OKIB_GATHER_PLAYERS, okib_list_string,
         NOIB_EMOJI_STRING, noib_list_string
     )
     #await ensure_display((_okib_channel.fetch_message(_okib_message_id)).edit, content=_list_content)
 
+async def check_almost_gather():
+    print(len(_okib_members)+round(0.1+len(_laterib_members)/2))
+    if len(_okib_members)+round(0.1+len(_laterib_members)/2) >= OKIB_GATHER_PLAYERS and not _gathered :
+        for member in _laterib_members:
+            await member.send("Hey, you are :laterib: and our radar indicates that the lobby gather is almost completed !! \nThis might be a great time for you to think about :okib: ;)")
 
 def gather_check():
+    global _gathered
     if len(_okib_members) >= OKIB_GATHER_PLAYERS and not _gathered:
         return True
         #ensure_display(functools.partial(combinator3000,(_okib_channel.fetch_message(_okib_message_id)).edit,gather,content=_list_content))) 
     if len(_okib_members) < OKIB_GATHER_PLAYERS and _gathered:
+        _gathered = False
         return False
-        
 
 async def up(ctx):
     global _okib_message_id
     
     if _okib_message_id is not None :
         await (await _okib_channel.fetch_message(_okib_message_id)).delete()
-    _okib_message_id = (await ctx.send(_list_content + '\n' +OKIB_GATHER_EMOJI_STRING)).id
-    _okib_message = await _okib_channel.fetch_message(_okib_message_id)
+
+    _okib_message = await ctx.send(_list_content)
```

Comment:
no ensuredisplay for these? it's fine for now I guess, since we'll need a few other little things to be 100% sure the code is multi-instance ready

Reply:
ignore this. I see you always call `up` in an ensuredisplay